posthog/utils.py

```diff
@@ -310,6 +310,8 @@ def load_data_from_request(request):
                 data_res["body"] = {**json.loads(request.body)}
             except:
                 pass
+        elif request.content_type == "text/plain":
```

Comment:
We have a ton of tests relating to encoding issues. Would be good to port those over for this.

Also not sure I understand text/plain if it's json?

Reply:
Some context for this is under https://github.com/PostHog/posthog-js/pull/144

> We have a ton of tests relating to encoding issues. Would be good to port those over for this.

Agreed. However I'd like to briefly roll this out, see whether this fixes session recording metrics or not for us and then do that. It might turn out that it doesn't help and we'll just roll this back or we might end up going for a different compression scheme.

Note also this change _should_ only affect our team: context under https://github.com/PostHog/posthog-js/pull/144

Reply:
Forgot to mention why text/plain for json: CORS. content-type: application/json is considered a custom header and results in every xhr doing 2 queries: OPTIONS + POST